__tests__/server.test.js

```diff
@@ -320,3 +319,25 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("DELETE /api/comments/:comment_id", () => {
+  test("should return a 204 code with no content on successful delete", () => {
+    return request(app).delete("/api/comments/6").expect(204);
+  });
+  test("should return 404 if the comment does not exist ", () => {
+    return request(app)
+      .delete("/api/comments/12576123")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.message).toBe("Comment does not exist");
+      });
+  });
+  test("should return 400 if the format of the id is wrong ", () => {
+    return request(app)
+      .delete("/api/comments/six")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.message).toBe("Invalid data type");
```

Comment:
I'd argue 'Bad request' is a better message here, less confusing,  because it might not be clear that 'invalid data type' is speaking about the path